src/sources/journald.rs

```diff
@@ -669,6 +681,10 @@ impl StartJournalctl {
             command.arg("--since=2000-01-01");
         }
 
+        if let Some(args) = &self.extra_journalctl_args {
+            command.arg(format!("{}", args));
+        }
```

Comment:
`command.arg` can only take a single argument. So, for example setting `extra_journalctl_args` to `--merge --utc` would come up as an unrecognised option.

It would be better to make `extra_journalctl_args` a `Vec<String>` so that multiple options can be specified.

Also specifying `format!("{}", args)` is unnecessary. You can just use `command.arg(args)` instead.

Reply:
Right, I changed to `Vec<String>`, please review.